hardware_interface/include/hardware_interface/handle.hpp

```diff
@@ -142,7 +143,9 @@ class Handle
 
   const std::string & get_prefix_name() const { return prefix_name_; }
 
-  [[deprecated("Use bool get_value(double & value) instead to retrieve the value.")]]
+  [[deprecated(
+    "Use std::optional<T> get_value() or T get_value(bool &status) or bool get_value(double & "
+    "value) instead to retrieve the value.")]]
```

Comment:
I think you forgot to update the deprecation message.

```suggestion
    "Use std::optional<T> get_value() or bool get_value(double & "
    "value) instead to retrieve the value.")]]
```